build.gradle

```diff
@@ -119,7 +119,13 @@ project('common') {
             project(':common').sourceSets.main.java
             project(':shared:protocol').sourceSets.main.java
         }
+        dependencies {
```

Comment:
I don't understand why these changes in `build.gradle` are showing up here, they are not part of this PR, right?